src/js/components/Provenance/Catalogue/CatalogueCard.tsx

```diff
@@ -0,0 +1,116 @@
+import { useMemo } from 'react';
+import i18n from '@/i18n';
+import { Card, Carousel, Descriptions, Flex, Space, Tag, Tooltip, Typography } from 'antd';
+import type { Project } from '@/types/metadata';
+import { isoDateToString } from '@/utils/strings';
+import { useTranslationFn } from '@/hooks';
+import { BOX_SHADOW } from '@/constants/overviewConstants';
+import Dataset from '@/components/Provenance/Catalogue/Dataset';
+
+const { Paragraph, Text, Title } = Typography;
+
+const MAX_KEYWORD_CHARACTERS = 50;
+
+const CatalogueCard = ({ project }: { project: Project }) => {
```

Comment:
I think for consistency this should be called Project. we can then later split it into multiple view modes, like with Dataset.

Reply:
I would rather change, Dataset to CatalogDataset as Dataset/Project by itself can mean multiple things in public context by conflicting with overview. I do understand that it is in a folder but the naming does reduce the readability.

Reply:
catalogdataset to me doesn't make sense, as the dataset isn't just used in the catalog

Reply:
I mean it in the form that a "Dataset" could mean multiple things while at least prefixing catalog tells the purpose. If you think otherwise still, lmk, I shall change it.